webpack.config.base.js

```diff
@@ -26,31 +26,43 @@ module.exports = {
       },
       {
         test: /\.(eot|svg|ttf|woff|woff2)$/,
-        loader: 'file-loader?name=[name].[ext]'
+        loader: 'file-loader',
+        options: {
+          'name': '[name].[ext]'
+        }
       },
       {
         test: /\.wav$/,
-        loader: 'file-loader?name=[name].[ext]'
+        loader: 'file-loader',
+        options: {
+          'name': '[name].[ext]'
+        }
       }
     ]
   },
   resolve: {
     extensions: ['.js', '.jsx', '.css', '.scss', '.json'],
     modules: [
       'node_modules'
-    ]
+    ],
+    fallback: {
+      "crypto": require.resolve('crypto-browserify'),
+      "stream": require.resolve("stream-browserify"),
+      buffer: require.resolve("buffer/"),
```

Comment:
Polyfills that were automatically brought in in webpack 4, but need to be specified in 5.